### PR TITLE
fix: streetAddress1, streetAddress2 field validation #12389

### LINKED ISSUE #12389: Bug: "streetAddress1", "streetAddress2" field validation
### What are you trying to achieve?

I want to standardize the validation messages

### Steps to reproduce the problem

Update address with too long streetAddress1 or streetAddress2 value: 

```
{
  "id": "VXNlcjoxMzY5",
  "input": {
    "city": "test",
    "cityArea": "test",
    "companyName": "test",
    "country": "US",
    "countryArea": "IN",
    "firstName": "test",
    "lastName": "test",
    "phone": "",
    "postalCode": "46802",
    "streetAddress1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum",
    "streetAddress2": ""
  }
}
```

Result: 
```
{
  "errors": [
    {
      "message": "Internal Server Error",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "addressUpdate"
      ],
      "extensions": {
        "exception": {
          "code": "KeyError"
        }
      }
    }
  ],
  "data": {
    "addressUpdate": null
  }
}
``` 

### What did you expect to happen?

Instead of Internal Server Error, a clear validation message that provided value is too long
e.g.:
```
    "addressUpdate": {
      "errors": [
        {
          "code": "INVALID",
          "field": "firstName",
          "addressType": null,
          "message": "Ensure this value has at most 256 characters (it has 447).",
          "__typename": "AccountError"
        },
```

### Logs

_No response_

### Environment

Saleor version: `core v3.13.0-a`
OS and version: …

### PR DESCRIPTION
**Pull request by sonbui00**
I want to merge this change because it fix #12389
streetAddress1, streetAddress2 validation does not work when update address

Fix #12389

### CODE REVIEW

**Review by pydeveloperashish — COMMENTED** [review]
This code change is working and giving the expected output.

**Review by aniav — COMMENTED** [review]
Hey @sonbui00 thank you for the changes.

A few comments:
1. Could you add a test that's addressing the error shown in #12389?
2. Please add a line in the changelog saying this issue has been fixed
3. There are tests failing in the `Pytest` action so basically you have to run tests locally as the change is actually breaking them

File: saleor/account/tests/test_account.py [hunk 1]
```diff
@@ -183,11 +208,6 @@ def test_copy_address(address):
     assert copied_address == address
 
 
-def test_compare_addresses(address):
-    copied_address = address.get_copy()
-    assert address == copied_address
```
Comment (aniav) [inline saleor/account/tests/test_account.py:188]: This test is covered by the test above so I removed it.

**Comment by aniav** [comment]
@sonbui00 I went ahead and added the missing changes to get this to a state where it can be reviewed and merged. Thank you for your contribution 👌

File: saleor/account/tests/test_account.py [hunk 2]
```diff
@@ -59,6 +60,30 @@ def test_address_form_postal_code_validation():
     assert "postal_code" in errors
 
 
+def test_address_form_long_street_address_validation():
```
Comment (IKarbowiak) [inline saleor/account/tests/test_account.py:63]: Missing `given, when, then`

**Review by IKarbowiak — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Review by jakubkuc — APPROVED** [review]

**Pull request merged**